code/__DEFINES/reagents.dm

```diff
@@ -184,12 +184,6 @@
 #define GIN 			"gin"
 #define ABSINTHE 			"absinthe"
 #define RUM 			"rum"
-#define TEQUILA 			"tequila"
-#define VERMOUTH 			"vermouth"
-#define WINE 			"wine"
-#define COGNAC 			"cognac"
-#define HOOCH 			"hooch"
-#define ALE 			"ale"
```

Comment:
Should I fix it in this PR? I guess I'll have to check back the older version to figure out what the deadrum variants were used for. What are deadrums by the way?